src/course-unit/add-component/ComponentIcon.jsx

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import { Icon } from '@edx/paragon';
+import { EditNote as EditNoteIcon } from '@edx/paragon/icons';
+
+import { componentTypeToIconMapping, COMPONENT_ICON_TYPES } from '../constants';
+
+const ComponentIcon = ({ type }) => {
+  const icon = componentTypeToIconMapping[type] || EditNoteIcon;
```

Comment:
[fyi] We added new icons in the [pull request for the fix](https://github.com/raccoongang/edx-platform/pull/2490/files#diff-cc20776c84f78638c79199d2b4d0c30c2e296b8dc82dc9b41de8a0706e0905edR183). Could you make sure that the lights will be displayed correctly for you?